v3.0/how-to/deploy/tidb-binlog.md

```diff
@@ -22,25 +22,18 @@ It is recommended to deploy TiDB Binlog using TiDB-Ansible. If you just want to
 
     | tidb-ansible branch | TiDB version | Note |
     | ------------------- | ------------ | ---- |
-    | release-2.0 | 2.0 version | The latest 2.0 stable version. You can use it in the production environment. |
-    | release-2.1 | 2.1 version | The latest 2.1 stable version. You can use it in the production environment (recommended). |
-    | master | master version | This version includes the latest features with a daily update. |
+    | release-3.0 | 3.0 stable | The latest 3.0 stable version. For use in production environments (recommended). |
+    | master | 3.0 unstable | This version includes the latest features with a daily update. |
 
 2. Use the following command to download the corresponding branch of TiDB-Ansible from the [TiDB-Ansible project](https://github.com/pingcap/tidb-ansible) on GitHub. The default folder name is `tidb-ansible`.
 
-    - Download the 2.0 version:
-        
-        ```bash
-        $ git clone -b release-2.0-new-binlog https://github.com/pingcap/tidb-ansible.git
-        ```
-
-    - Download the 2.1 version:
+    - Download the 3.0 branch:
 
         ```bash
-        $ git clone -b release-2.1 https://github.com/pingcap/tidb-ansible.git
+        $ git clone -b release-3.0 https://github.com/pingcap/tidb-ansible.git
```

Comment:
Now the tidb and binlog versions are corresponding, so need to download the tidb-ansible of the specified tag. 
`git clone -b $tag https://github.com/pingcap/tidb-ansible.git`
If already have tidb-ansible, you can use it directly to deploy the binlog.